app/src/main/java/com/compscieddy/meetinthemiddle/HomeActivity.java

```diff
@@ -60,12 +60,18 @@
   private GoogleApiClient mGoogleApiClient;
   private Marker mCurrentMarker;
 
-  @Bind(R.id.map) MapView mMapView;
```

Comment:
Put all on same line - I know, it's annoying how Android Studio automatically adds a new line